add-on/chrome/bootstrap.js

```diff
@@ -481,11 +481,64 @@ var WindowListener = {
         gBrowser.tabContainer.removeEventListener("TabSelect", this);
         gBrowser.removeEventListener("DOMTitleChanged", this);
         gBrowser.removeEventListener("mousemove", this);
+        this.removeRemoteCursor();
         this._listeningToTabSelect = false;
         this._browserSharePaused = false;
       },
 
       /**
+       *  If sharing is active, paints and positions the remote cursor
+       *  over the screen
+       *
+       *  @param cursorData Object with the correct position for the cursor
+       *                    {
+       *                      ratioX: position on the X axis (percentage value)
+       *                      ratioY: position on the Y axis (percentage value)
+       *                    }
+       */
+      addRemoteCursor: function(cursorData) {
+        if (!this._listeningToTabSelect) {
+          return;
+        }
+
+        let browser = gBrowser.selectedBrowser;
+        let cursor = browser.parentNode.querySelector("#remote-cursor");
+
+        if (!cursor) {
+          cursor = document.createElement("image");
+          cursor.setAttribute("id", "remote-cursor");
+          cursor.style.pointerEvents = "none";
```

Comment:
Move the style to `add-on/chrome/skin/shared/loop.css` with the rest of the styles. And let's rename this to #loop-remote-cursor as it's in the global browser space.